`go get` broken due to compile errors referring to complete package

```
go get -u github.com/grafana/tanka/cmd/tk                                                                                                                                  ⏎
# github.com/grafana/tanka/pkg/cli/cmp
../../../go/src/github.com/grafana/tanka/pkg/cli/cmp/complete.go:27:9: undefined: complete.PredictNothing
../../../go/src/github.com/grafana/tanka/pkg/cli/cmp/complete.go:40:21: undefined: complete.PredictDirs
../../../go/src/github.com/grafana/tanka/pkg/cli/cmp/complete.go:48:23: undefined: complete.Flags
../../../go/src/github.com/grafana/tanka/pkg/cli/cmp/complete.go:67:22: undefined: complete.Commands
../../../go/src/github.com/grafana/tanka/pkg/cli/cmp/complete.go:70:27: cannot use Create(c) (type complete.Command) as type *complete.Command in assignment
```

I have tried both running the command outside of GOPATH, and also unsetting GOPATH, both to no avail:

```
dieter@dieter-m6800 cd /tmp
dieter@dieter-m6800 tmp go get -u github.com/grafana/tanka/cmd/tk
# github.com/grafana/tanka/pkg/cli/cmp
/home/dieter/go/src/github.com/grafana/tanka/pkg/cli/cmp/complete.go:27:9: undefined: complete.PredictNothing
/home/dieter/go/src/github.com/grafana/tanka/pkg/cli/cmp/complete.go:40:21: undefined: complete.PredictDirs
/home/dieter/go/src/github.com/grafana/tanka/pkg/cli/cmp/complete.go:48:23: undefined: complete.Flags
/home/dieter/go/src/github.com/grafana/tanka/pkg/cli/cmp/complete.go:67:22: undefined: complete.Commands
/home/dieter/go/src/github.com/grafana/tanka/pkg/cli/cmp/complete.go:70:27: cannot use Create(c) (type complete.Command) as type *complete.Command in assignment
dieter@dieter-m6800 tmp unset GOPATH
dieter@dieter-m6800 tmp go get -u github.com/grafana/tanka/cmd/tk
# github.com/grafana/tanka/pkg/cli/cmp
/home/dieter/go/src/github.com/grafana/tanka/pkg/cli/cmp/complete.go:27:9: undefined: complete.PredictNothing
/home/dieter/go/src/github.com/grafana/tanka/pkg/cli/cmp/complete.go:40:21: undefined: complete.PredictDirs
/home/dieter/go/src/github.com/grafana/tanka/pkg/cli/cmp/complete.go:48:23: undefined: complete.Flags
/home/dieter/go/src/github.com/grafana/tanka/pkg/cli/cmp/complete.go:67:22: undefined: complete.Commands
/home/dieter/go/src/github.com/grafana/tanka/pkg/cli/cmp/complete.go:70:27: cannot use Create(c) (type complete.Command) as type *complete.Command in assignment
```

--- a/docs/docs/installation.md
+++ b/docs/docs/installation.md
@@ -28,10 +28,10 @@ Just grab the latest version from there, download it and put somewhere in your
 ## From source
 
 In case the above won't work for you, you can try building the most recent
-code using `go get`:
+release using `go get`:
 
 ```bash
-$ go get -u github.com/grafana/tanka/cmd/tk
+$ GO111MODULE=on go get github.com/grafana/tanka/cmd/tk
 ```
 
 If that won't work either, compile by hand:

